### ISSUE TITLE
Asset Tag generation for Components

### ISSUE THREAD

**Issue by i-am-checking** [grokability/snipe-it#14726]
### Is your feature request related to a problem? Please describe.

I've been trying to find the best way to attach asset tags to monitors
We need to know how many we have, who has dual and single monitors, and the number of different models, so assigning them to assets is a key part of the functionality we need. 
Being able to asset tag them and stickering them is key for easily being able to manage them in the case of a failure. Being a school, we get a lot of vandals in IT suites so being able to quickly and easily mark is as dead and replace it is key

### Describe the solution you'd like

I'd like to be able to have names and asset tags against components, even for things like GPU's, having individual names against them would make it easier to keep track of and easier to change status on the go.

### Describe alternatives you've considered

The only way we are able to do this currently is putting them down as assets, and assigning the location to the room they are in, but we have found it is not specific enough.
Being able to quickly identify which PC it is attached to saves time and makes it easier to keep track of

**Comment by snipe** [comment]
A monitor wouldn't really be a component in the first place tbh, better served as an accessory - but also, only assets have asset tags within Snipe-IT, as you rightly stated. 

We do not have plans to add asset tags to components at this time. 

It sounds like perhaps an inverse relationship in the display/API might be the thing that could get you closer. For example, on an asset detail page, we have a tab for "Assets" which are the assets that the asset you're looking at have assigned to them. Perhaps something more of an "is parent asset of" tab that would show which assets consider the asset you're looking at as their "parent" or owner?

**Comment by i-am-checking** [comment]
Yes a "parent of asset" tab would be the best solution for this I think

**Comment by i-am-checking** [comment]
Can this be added to the CSV uploader potentially? I can check them out manually but with about 1000 monitors and touch displays across our schools it would be incredibly time consuming
